themes-default/slim/views/addShows_addExistingShow.mako

```diff
@@ -14,8 +14,87 @@ const startVue = () => {
         },
         data() {
             return {
```

Comment:
Please add `metaInfo: { title: 'TITLE_HERE' }` replacing `TITLE_HERE` with the correct title.

Reply:
Exists on lines 12-14.
(I rebased on current develop)